File: src/main/java/com/xavierbouclet/whiskies/api/service/WhiskyService.java
package com.xavierbouclet.whiskies.api.service;

import com.xavierbouclet.whiskies.api.model.Whisky;
import org.springframework.web.service.annotation.GetExchange;
import reactor.core.publisher.Flux;

public interface WhiskyService {

    @GetExchange("/whiskies")
    Flux<Whisky> loadAll();
}

File: src/main/kotlin/com/xavierbouclet/whiskies/api/service/WhiskyService.kt
package com.xavierbouclet.whiskies.api.service

import com.xavierbouclet.whiskies.api.model.Whisky
import org.springframework.web.service.annotation.GetExchange

interface WhiskyService {
    @GetExchange("/whiskies")
    fun loadAll(): List<Whisky>
}